docs/layouts/shortcodes/generated/cluster_configuration.html

```diff
@@ -70,7 +70,7 @@
         </tr>
         <tr>
             <td><h5>cluster.thread-dump.stacktrace-max-depth</h5></td>
-            <td style="word-wrap: break-word;">8</td>
+            <td style="word-wrap: break-word;">24</td>
```

Comment:
Hi @Myasuka and @yuchen-ecnu , thanks for your contribution. It's a great proposal, the 8 is too small for troubleshooting.

However, I'm afraid the 24 is also too small, could you add some information about why you set it to 24? I may have missed some information.

we set it to 80 in my company. Because some stacks are too deep, like this: 

![image](https://user-images.githubusercontent.com/38427477/207609774-c373784c-ebaa-40f1-a985-3d50254668d0.png)

Reply:
Actually, we can set this value as 10000 or even larger to make sure we can see the whole stack in any case. However, this might also increase the whole result size of the returned API. The current suggestion of 24 is just based on our experiences in the production environment. I think it's fine to increase the default value to a larger one with your input.

@yuchen-ecnu WDYT?

Reply:
Hi @1996fanrui , @Myasuka.

I think such a deep call stack might not be required for most scenarios, and in our experience so far 24 seems to be a value that can cover most cases. 

In my opinion, to balance the performance of the Rest API and analysis experience, we can consider setting up a deeper call stack with the parameters provided by Flink for such a job.

https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/#cluster-thread-dump-stacktrace-max-depth

Reply:
Hi @Myasuka @yuchen-ecnu , thanks for your feedback.

I helped our users troubleshoot some problems this week and found that the stack depth of flink Task threads can easily exceed 24. As shown in the figure: there are only three lines of business code, and nearly 40 lines of flink code.

If a Task contains many operators, the stack depth will be deeper. How about 40 or 50?

![image](https://user-images.githubusercontent.com/38427477/208626712-55338df8-929e-4d00-b5e1-dbd43ffad707.png)

Reply:
Hi @1996fanrui @Myasuka, 
With the two instances shared by @1996fanrui , I think it is necessary to further increase the default depth of Thread Dump. How about increasing this value to 50?

Reply:
@yuchen-ecnu I think it's okay, let's change the default value and the related commit messages.